.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## [Ticket](https://wicmtdp.atlassian.net/browse/{TICKET_ID})
```

Comment:
Could I suggest this change instead? I find it confusing when there are headings with no content and I think it's a good idea to explicitly list the ticket. It also allows for PRs that manage to hit multiple tickets.

```suggestion
## Ticket
- https://wicmtdp.atlassian.net/browse/{TICKET_ID}
```